annotator-core/src/main/java/edu/ucr/cs/riple/core/adapters/NullAwayV1Adapter.java

```diff
@@ -57,8 +56,8 @@ public class NullAwayV1Adapter implements NullAwayVersionAdapter {
    */
   private int offset;
 
-  public NullAwayV1Adapter(Config config) {
-    this.config = config;
+  public NullAwayV1Adapter(Config config, FieldDeclarationStore fieldDeclarationStore) {
+    super(config, fieldDeclarationStore, 0);
```

Comment:
Shouldn't this pass `1` for the version?

Reply:
4064cd61fde7ae37dcca41407d2f1e306be1198e